core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-edit-content-category-field-dialog/dot-edit-content-category-field-dialog.component.scss

```diff
@@ -0,0 +1,37 @@
+@use "variables" as *;
+
+:host {
+    display: block;
+    height: 100%;
+
+    .category-field__dialog {
```

Comment:
Don't need the nesting, the encapsulation takes care of it.

Reply:
improved.